opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -206,7 +206,7 @@ struct MaxSinglePrecisionDays<TypeTag, TTag::FlowModelParameters> {
 };
 template<class TypeTag>
 struct MaxStrictIter<TypeTag, TTag::FlowModelParameters> {
-    static constexpr int value = 8;
+    static constexpr int value = 0;
```

Comment:
This raised my eyebrows... No strict iterations seems very dangerous, even though the relaxed max fraction is low. Your testing makes me relax (ha...) a bit though. Did you try to use a somewhat stricter value for the ToleranceCnvRelaxed? Such as 0.2 or 0.1?

Reply:
I have not looked into the implementation, but basically I was of the impression that combining it with RelaxedMaxPvFraction basically means all iterations are strict. The relaxed tolerance only applies to 3% of the pore volume. Moreover, the relaxed tolerance is tightened from 1e9 to 1. In effect, I believe these are far stricter settings than before. Yes I have played with even lower values for the relaxed tolerance, including 0.1 and 0.4. In some cases it is detrimental to performance (computational time almost doubling).

Reply:
> I have not looked into the implementation, but I was of the impression that combining it with RelaxedMaxPvFraction basically means all iterations are strict. Specifically, that the relaxed tolerance here only applies to 3% of the pore volume. 

Your understanding is correct. One could envision putting a small number for MaxStriciter and combine it with the RelaxedMaxPvFraction. Then the relaxation of the tolerance for  3%  of the pore volume will only happen when iter>maxStrictIter. I have not yet seen any cases where this is necessary. I can also add that I have tested these and similar options for many simulations during the last year. And it seems to work as long as the relaxed tolerance is ~1. 

@alfbr Have you checked the results for the failing tests?

Reply:
> @alfbr Have you checked the results for the failing tests?

No I have not, I will take  a look.